bin/i18n/i18n_script_utils.rb

```diff
@@ -217,8 +217,8 @@ def self.write_markdown_with_header(markdown, header, path)
   #
   # Right now, this is just page titles but it could be expanded to include
   # any English content (description, social share stuff, etc).
-  def self.sanitize_header!(header)
-    header.slice!("title")
+  def self.sanitize_markdown_header(header)
+    header.slice('title')
```

Comment:
curious why we're not modifying header anymore?

Reply:
We do modify but in a more understandable way
https://github.com/code-dot-org/code-dot-org/blob/8bead65f17ada3369031f1f71dd9ec450ded7d4d/bin/i18n/resources/pegasus/hourofcode/sync_in.rb#L47-L49

Reply:
@annaxuphoto Mario has already tested this, but if you want to do it by yourself, you need:
1. Run `sync-down`: `bundle exec bin/i18n/sync-down.rb`
2. Check that a Crowdin locale dir has a HOC translation file: `i18n/locales/{Spanish,Italian,etc.}/hourofcode/**/*.md`
3. Run `sync-out` for the HOC resource: `bundle exec bin/i18n/resources/pegasus/hourofcode/sync_out.rb`
4. Check that the translations are distributed into the dir `pegasus/sites.v3/hourofcode.com/i18n` - files inside should be updated with new Crowdin translations

Reply:
thanks!